sdk/communication/azure-communication-callautomation/src/main/java/com/azure/communication/callautomation/CallMediaAsync.java

```diff
@@ -443,4 +445,111 @@ private PlaySourceInternal getPlaySourceInternalFromRecognizeOptions(CallMediaRe
+    }
+
+    Mono<Response<Void>> sendDtmfWithResponseInternal(CommunicationIdentifier targetParticipant, List<DtmfTone> tones, String operationContext, Context context) {
+        try {
+            context = context == null ? Context.NONE : context;
+            SendDtmfRequestInternal requestInternal = new SendDtmfRequestInternal()
+                .setTargetParticipant(CommunicationIdentifierConverter.convert(targetParticipant))
+                    .setTones(tones.stream()
+                .map(this::convertDtmfToneInternal)
+                .collect(Collectors.toList()))
+                        .setOperationContext(operationContext);
+
+            return contentsInternal.sendDtmfWithResponseAsync(callConnectionId, requestInternal, context);
+        } catch (RuntimeException e) {
+            return monoError(logger, e);
+        }
+    }
+
+
+    /**
+     * Starts continuous Dtmf recognition.
+     *
+     * @param targetParticipant the target participant
+     * @param operationContext operationContext (pass null if not applicable)
+     * @return void
+     */
+    public Mono<Void> startContinuousDtmfRecognition(CommunicationIdentifier targetParticipant, String operationContext) {
+        return startContinuousDtmfRecognitionWithResponse(targetParticipant, operationContext).then();
+    }
+
+    /**
+     * Starts continuous Dtmf recognition.
+     * @param targetParticipant the target participant
+     * @param operationContext operationContext (pass null if not applicable)
+     * @return Response for successful start continuous dtmf recognition request.
+     */
+    public Mono<Response<Void>> startContinuousDtmfRecognitionWithResponse(CommunicationIdentifier targetParticipant, String operationContext) {
+        return withContext(context -> startContinuousDtmfRecognitionWithResponseInternal(targetParticipant, operationContext, context));
+    }
+
+    Mono<Response<Void>> startContinuousDtmfRecognitionWithResponseInternal(CommunicationIdentifier targetParticipant, String operationContext, Context context) {
+        try {
+            context = context == null ? Context.NONE : context;
+            ContinuousDtmfRecognitionRequestInternal requestInternal = new ContinuousDtmfRecognitionRequestInternal()
+                .setTargetParticipant(CommunicationIdentifierConverter.convert(targetParticipant))
+                .setOperationContext(operationContext);
+
+            return contentsInternal.startContinuousDtmfRecognitionWithResponseAsync(callConnectionId, requestInternal, context);
+        } catch (RuntimeException e) {
+            return monoError(logger, e);
+        }
+    }
+
+    /**
+     * Stops continuous Dtmf recognition.
+     * @param targetParticipant the target participant
+     * @param operationContext operationContext (pass null if not applicable)
+     * @return void
+     */
+    public Mono<Void> stopContinuousDtmfRecognition(CommunicationIdentifier targetParticipant, String operationContext) {
```

Comment:
Remove optional parameter, only keep it in "WithResponse" method.

Reply:
done, thanks!